e2e-tests/tfm/cypress/e2e/journeys/case-amendments/add-bank-decision-proceed.spec.js

```diff
@@ -101,8 +101,8 @@ context('Amendments underwriting - add banks decision - proceed', () => {
     cy.url().should('contain', 'cover-end-date');
 
     amendmentsPage.amendmentCoverEndDateDayInput().clear().focused().type(dateConstants.tomorrowDay);
-    amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
-    amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
+    amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.tomorrowMonth);
```

Comment:
Will this fail when running tomorrow?
I think I have seen something similar before where an E2E was failing on a specific date.

Reply:
No - line 103 uses the `tomorrowDay` constant, but the previous 104 and 105 were using `todayMonth` and `todayYear` which explains why it had been passing every day up to now